src/EPiServer.SocialAlloy.Web/Social/Tutorials/LikeButton/LikeButtonBlockController.cs

```diff
@@ -0,0 +1,150 @@
+﻿using EPiServer.ServiceLocation;
+using EPiServer.Social.Common;
+using EPiServer.Social.Ratings.Core;
+using EPiServer.SocialAlloy.Web.Social.Blocks;
+using EPiServer.SocialAlloy.Web.Social.Models;
+using EPiServer.Web.Mvc;
+using EPiServer.Web.Routing;
+using Microsoft.AspNet.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+
+namespace EPiServer.SocialAlloy.Web.Social.Controllers
+{
+    /// <summary>
+    /// The LikeButtonBlockController is a simple implementation of a Like button using
+    /// the EPiServer Social Ratings service API.  In this simplistic implementation we 
+    /// assume that if the user is not logged in that some anonymous user can like the page
+    /// indefinitely; not practical but should help to illustrate rating statistics
+    /// as well as keep things simpler without going into managing user authentication.
```

Comment:
I don't think we need to justify our ignoring user auth here. It may just confuse them. I can see a future tutorial on working with users...

How about just...

> The LikeButtonBlockController is a simple implementation of a Like button using
> the EPiServer Social Ratings service API.  In this implementation we assume that 
> if the user is not logged in (anonymous), they can "like" the page as many times
> as they choose.